WebRoot/messages/ZmMsg_it.properties

```
@@ -730,7 +730,7 @@ confirmDeclineAppt = Rifiutare l\u2019appuntamento?
 confirmDeleteCalendar = Vuoi davvero eliminare definitivamente l\u2019Agenda "{0}"?
 confirmDeleteContact = Vuoi davvero eliminare questo contatto?
 confirmDeleteFolder = Vuoi davvero eliminare definitivamente la cartella {0}?
-confirmDeleteForever = Eliminare in modo permanente gli {0,choice,1#elemento|2#elementi} selezionati?
+confirmDeleteForever = Eliminare in modo permanente {0,choice,1#l'elemento selezionato|2#gli elementi selezionati}?
```

had to make below adjustment to show it properly in UI
```suggestion
confirmDeleteForever = Eliminare in modo permanente {0,choice,1#l\u2019elemento selezionato|2#gli elementi selezionati}?
```